src/ProjectFileTools/GoToDefinition/GoToDefinitionController.cs

```diff
@@ -41,11 +44,55 @@ public int QueryStatus(ref Guid pguidCmdGroup, uint cCmds, OLECMD[] prgCmds, Int
             return Next?.QueryStatus(pguidCmdGroup, cCmds, prgCmds, pCmdText) ?? (int)Microsoft.VisualStudio.OLE.Interop.Constants.OLECMDERR_E_NOTSUPPORTED;
         }
 
+        private static bool PackageExistsOnNuGet(string packageName, string version, out string url)
+        {
+            string packageAndVersionUrl = $"https://www.nuget.org/packages/{packageName}/{version}/";
+            string packageUrl = $"https://www.nuget.org/packages/{packageName}/";
+
+            try
+            {
+                using (HttpClient client = new HttpClient())
+                {
+                    HttpResponseMessage response = client.GetAsync(packageAndVersionUrl).Result;
+
+                    if (response.StatusCode == System.Net.HttpStatusCode.OK)
+                    {
+                        url = packageAndVersionUrl;
+                        return true;
+                    }
+                    else
+                    {
+                        response = client.GetAsync(packageAndVersionUrl).Result;
```

Comment:
Why are you fetching the same URL again? Isn't this supposed to be `packageUrl`?